### ISSUE TITLE
Getting P is not defined

### ISSUE THREAD

**Issue by cranesandcaff** [jrudio/meteor-bluebird#3]
P isn't defined on the server but it is showing up on the client.

**Comment by jrudio** [comment]
That is correct per [Bluebird's website](http://bluebirdjs.com/docs/getting-started.html)

Under the `browser` section of that link:
>When using script tags the global variables Promise and P (alias for Promise) become available. Bluebird runs on a wide variety of browsers including older versions.

`P` won't be available on the server, I need to update the readme as it doesn't reflect this.

I believe the regular `Promise` is available on the server. Would making `P` available on the server be beneficial?
